x-pack/plugins/cases/public/components/create/index.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiButtonEmpty, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import styled from 'styled-components';
+
+import { Field, getUseField } from '../../common/shared_imports';
+import * as i18n from './translations';
+import { CreateCaseForm } from './form';
+import { FormContext } from './form_context';
+import { SubmitCaseButton } from './submit_button';
+import { Case } from '../../containers/types';
+
+export const CommonUseField = getUseField({ component: Field });
+
+const Container = styled.div`
+  ${({ theme }) => `
+    margin-top: ${theme.eui.euiSize};
+  `}
+`;
+
+export interface CreateCaseProps {
```

Comment:
Do you think it make sense to export also [`afterCaseCreated`](https://github.com/elastic/kibana/pull/94444/files#diff-05700ace35e1262b30167f4cd76f3a111c05d51ec380ef263f471fa40ac5d4d3R34)? It is a callback that is being called after the creation of the case but before the push of the case. `onSuccess` is being called after the creation and the push of the case.

Reply:
the link didn't jump to the code. can you show me?